openpuc_scrapers/scrapers/ny.py

```diff
@@ -61,8 +45,7 @@ class NYPUCDocketInfo(BaseModel):
     industry_affected: str
 
```

Comment:
Missing fields ([see these docs](https://docs.google.com/spreadsheets/d/1aoV7W-6JD8GRk5NsbW8y6hmtsvaLYV2pOV8P80f9UJc/edit?gid=0#gid=0)):

- `organization: str` (though maybe it should be `OrganizationNY`)?
- `matter_type: str`
- `matter_subtype: str`
- `case_title: str`
- `related_cases: list[DocketNY]`
- `party_list: list[OrganizationNY]` (or `list[PartyNY]`]?
- `calendar: list[EventNY]`?